src/app/components/dialogs/chartDialog/chartDialog.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { DialogData } from '../baseDialogService.abstract';
+
+export interface ChartDialogData {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  content: object;
+}
+
+@Component({
+  selector: 'app-chart-dialog',
+  templateUrl: './chartDialog.component.html',
+  styleUrls: ['./chartDialog.component.scss']
+})
+export class ChartDialogComponent implements OnInit {
+  public content = {};
+  constructor(@Inject(MAT_DIALOG_DATA) public dialogData: DialogData<ChartDialogData>) { }
+
+  ngOnInit(): void {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Consequence of using a generic object is that we no longer get the benefits of Typescript, plus having to add exceptions everywhere its used